pygaggle/rerank/transformer.py

```diff
@@ -30,32 +31,64 @@
            'QuestionAnsweringTransformerReranker',
            'SentenceTransformersReranker']
 
+prediction_tokens = {
+        'castorini/monot5-base-msmarco':          ['▁false', '▁true'],
+        'castorini/monot5-base-msmarco-10k':      ['▁false', '▁true'],
+        'castorini/monot5-large-msmarco':         ['▁false', '▁true'],
+        'castorini/monot5-large-msmarco-10k':     ['▁false', '▁true'],
+        'castorini/monot5-base-med-msmarco':      ['▁false', '▁true'],
+        'castorini/monot5-3b-med-msmarco':        ['▁false', '▁true'],
+        'unicamp-dl/ptt5-base-msmarco-pt-10k':     ['▁não'  , '▁sim'],
+        'unicamp-dl/ptt5-base-msmarco-pt-100k':    ['▁não'  , '▁sim'],
+        'unicamp-dl/ptt5-base-msmarco-en-pt-10k':  ['▁não'  , '▁sim'],
+        'unicamp-dl/mt5-base-multi-msmarco':       ['▁no'   , '▁yes'],
+        'unicamp-dl/mt5-base-en-pt-msmarco':       ['▁no'   , '▁yes']
+        }
+
 
 class MonoT5(Reranker):
-    def __init__(self,
-                 model: T5ForConditionalGeneration = None,
-                 tokenizer: QueryDocumentBatchTokenizer = None,
-                 use_amp = False):
-        self.model = model or self.get_model()
-        self.tokenizer = tokenizer or self.get_tokenizer()
+    def __init__(self, 
+                 pretrained_model_name_or_path: str  = 'castorini/monot5-base-msmarco',
+                 use_amp = False,
+                 token_false = None,
+                 token_true  = None):
+        self.model = self.get_model(pretrained_model_name_or_path)
+        self.tokenizer = self.get_tokenizer(pretrained_model_name_or_path)
+        self.token_false, self.token_true = token_false or self.get_prediction_tokens(
```

Comment:
this line seems wrong as `self.token_true` is receiving the value of `token_false`?